specification/communication/data-plane/Chat/preview/2023-11-15-preview/examples/Messages_ListChatMessagesWithPageSize.json

```diff
@@ -11,19 +11,33 @@
         "value": [
           {
             "id": "1593107077690",
-            "type": "text",
+            "type": "html",
             "version": "1593107077683",
             "sequenceId": "5",
             "content": {
-              "message": "So where should we get lunch from today?"
+              "message": "<p>This place for lunch?&nbsp;</p>\r\n<p><img itemscope=\"png\" src=\"\" width=\"1425\" height=\"539\" alt=\"image\" id=\"0-canaryeus-d1-907ca0083e8f3bef6793e5a46c0ded73\"></p>",
+              "attachments": [
+                {
+                  "id": "0-canaryeus-d1-907ca0083e8f3bef6793e5a46c0ded73",
+                  "attachmentType": "teamsInlineImage",
```

Comment:
for 11-15, we have renamed this to `image`, also `contentType` is removed. And gateway endpoint has been updated to `chat/thread/<thread_id>/images/<image_id>/views/<view_name>`